README.md

```diff
@@ -165,7 +168,7 @@ If you work in an office, any space where you will leave behind computer or netw
 
 ### Computers
 
-If you are working for Rapid River on your own computing equipment, it must run one of the following operating systems:
+If you are working for Rapid River on your computing equipment, it must run one of the following operating systems:
```

Comment:
`your own` is an intensifier that is common in American English. Intended; please keep.